templates/swagger/v1_json.tmpl

```diff
@@ -17585,6 +17585,41 @@
       },
       "x-go-package": "code.gitea.io/gitea/modules/structs"
     },
+    "ContributorData": {
```

Comment:
I think we should move these types out of `modules/structs`.
For now, they are not needed here as they only pollute the API.

Reply:
Well…
if they are only used there, we can move them exactly into `/services/repository/contributors_graph.go` for now.
That's at least the easiest solution in my mind.

Reply:
Fixed in [0f61e59](https://github.com/go-gitea/gitea/pull/27882/commits/0f61e592e00ef57ae8c748fbec709d4edeeab637)